README.md

```diff
@@ -73,6 +73,36 @@ Most importantly this crate does not deal with authority set changes. It assumes
 authorities is always the same. Authority set handoffs are handled in Substrate by listening to
 signals emitted on the underlying blockchain.
 
+### Fuzzing
+
+To run the fuzzing test harness you need to install either `afl` or `cargo-fuzz`:
+
+```sh
+cargo install cargo-fuzz
+cargo install afl
+```
+
+#### libfuzzer
+
+```sh
+cargo fuzz run graph
+cargo fuzz run round
```

Comment:
This might require nightly rustc? Is that worth mentioning, or perhaps a note about it is expected to be obsolete soon anyway?

Reply:
Good point, I only use nightly so didn't notice it. Updated the readme.